common/utils/src/main/resources/error/error-classes.json

```diff
@@ -92,6 +92,13 @@
       "The method <methodName> can not be called on streaming Dataset/DataFrame."
     ]
   },
+  "CANNOT_ALTER_PARTITION_COLUMN" : {
+    "message" : [
+      "ALTER TABLE CHANGE COLUMN is not supported for partition columns in non-delta tables.",
```

Comment:
Does Spark support the `... CHANGE COLUMN` command? Looking at the test, it should be `ALTER TABLE ... ALTER COLUMN`

Reply:
I believe that `ALTER COLUMN` and `CHANGE COLUMN` are interchangeable based on the parser rules https://github.com/apache/spark/blob/c8a209251374c28868aea809f51c467fdd0d1443/sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4#L129